com.unity.multiplayer.mlapi/Tests/Editor/BitStreamTests.cs

```diff
@@ -794,5 +794,21 @@ public void TestStringPackedDiff()
             Assert.That(readBuilder.ToString(), Is.EqualTo(testString));
             Assert.That(stringCompare.ToString(), Is.EqualTo(testString));
         }
+
+        [Test]
+        public void TestSerializationPipelineBool()
+        {
+            NetworkBuffer buffer = new NetworkBuffer();
+            NetworkWriter writer = new NetworkWriter(buffer);
+            NetworkReader reader = new NetworkReader(buffer);
+
+            writer.WriteObjectPacked(true);
+            writer.WriteObjectPacked(false);
```

Comment:
you might add other flavors of types here such as `int[]`, `INetworkSerializable`, `Vector3` etc to test a few other things too.

Reply:
Yes, this PR is just for the regression of the bool bug though. But I will test all the packed ones.

Reply:
what about `bool[]` then? :)

Reply:
ah, also Yamato fails because fix #541 went into `release/0.1.0` and it's not on `develop` branch yet. so the test fails, which is very good! (except 2021.1 on Win passed? oh no...)

Reply:
https://yamato.cds.internal.unity3d.com/jobs/1201-Unity%20MLAPI/tree/test%252Fregression-write-object-packed-bool/.yamato%252Fproject-test.yml%2523test_project_com.unity.multiplayer.mlapi_ubuntu_trunk/5704466/job/full-pipeline

Looks failed?

Reply:
And yes, I put int he PR description: "Expect this to fail until the fix for this has been backported to develop."

Reply:
Yea as for yamato. The tests failed on all machine. But pack and validate succeeded. Which is a good sign!

Reply:
Sure